README.md

```diff
@@ -1,2 +1,16 @@
-# vehicle-app-template
-vehicle-app-template
+# Vehicle App Template
+
+The repository to get you started with Velocitas App development. It contains a very basic setup which will walk you through assisted project creation.
+
+## Required software
+
+* Visual Studio Code
+* A container runtime supported by Visual Studio Code's devContainer feature. See [here](https://eclipse.dev/velocitas/docs/tutorials/quickstart/container_runtime/)
+
+## Setup steps
+
+1. Clone the repository **or** use Github Codespaces
+2. Open the repository in Visual Studio Code **or** in Github Codespaces
+3. Follow the steps through the assisted project creation
```

Comment:
should we provide some more details about the creation process and what can be selected?

Reply:
The steps in the CLI should be assisted enough to be self explanatory. If we describe them here, it will be outdated rather soon.